guides/source/getting_started.md

```diff
@@ -1703,7 +1703,9 @@ app/controllers/concerns
 app/models/concerns
 ```
 
-A given blog article might have various statuses - for instance, it might be visible to everyone (i.e. `public`), or only visible to the author (i.e. `private`). It may also be hidden to all but still retrievable (i.e. `archived`). Comments may similarly be hidden or visible. This could be represented using a `status` column in each model.
+In the example below, we will create a new feature for our blog that does not use concerns, but would benefit from them. Then, we will do a refactor where we begin using concerns in order to make our code more maintainable and reusable.
```

Comment:
I toyed with the wording a bit to make it more concise.  Does the following still address the point of confusion sufficiently?

```suggestion
In the example below, we will implement a new feature for our blog that would benefit from using a concern. Then, we will create a concern, and refactor the code to use it, making the code more DRY and maintainable.
```

(Note that "DRY" has already been introduced [at the start of the guide](https://guides.rubyonrails.org/v7.0.0/getting_started.html#what-is-rails-questionmark).)

Reply:
Yes it does!